src/main/java/wooteco/subway/line/domain/Sections.java

```diff
@@ -140,4 +154,27 @@ public void removeStation(Station station) {
         upSection.ifPresent(it -> sections.remove(it));
         downSection.ifPresent(it -> sections.remove(it));
     }
+
+    public Station startStation() {
+        return orderedStations.get(0);
+    }
+
+    public Station endStation() {
+        int size = orderedStations.size();
+        return orderedStations.get(size - 1);
+    }
+
+    public int calculateTotalDistance() {
+        return getSections().stream()
+            .mapToInt(Section::getDistance)
+            .sum();
```

Comment:
```suggestion
        return sections.stream()
            .mapToInt(Section::getDistance)
            .sum();
```

그냥 `sections`를 사용해도 되는 상황이네요!